docker/helm.version

```diff
@@ -0,0 +1,4 @@
+# NB Helm clients will refuse to play with Tiller that is older. 2.8.2
+# is the first release that had checksums, so it's used.
+HELM_VERSION=2.8.2
```

Comment:
You might want to look into upgrading the helm client to 2.9.0 at least according to: [these artifactory docs](https://www.jfrog.com/confluence/display/RTF/Helm+Chart+Repositories#HelmChartRepositories-ConfiguringtheHelmClient). From what I've seen from the rest of this PR, this is only for testing locally on the container if you choose to? I think if you are testing, having auth working would still be desired.

I could be way off on this one though, I'm fairly new to this repo! Figured I would point it out though.

Reply:
Good point @mellena1 -- I hadn't clocked that version difference. We can specify a minimum version of Helm/Tiller of 2.9 (if it's been out for a while at least).

Reply:
Note to self: probably need to bump the Helm client package version too.

Reply:
Sounds good 😄